mysql/migrations.sql

```diff
@@ -89,3 +89,19 @@ ALTER TABLE `taskHistory` ADD KEY `runId` (`runId`, `requestId`);
 ALTER TABLE `requestHistory` ADD COLUMN message VARCHAR(280) NULL;
 
 ALTER TABLE `deployHistory` ADD COLUMN message VARCHAR(280) NULL;
+
+--changeset wsorenson:10 dbms:mysql
+ALTER TABLE `taskHistory`
+  ADD COLUMN `host` VARCHAR(100) CHARACTER SET ASCII NULL,
+  ADD COLUMN `startedAt` TIMESTAMP NULL,
```

Comment:
we lose precision here (milliseconds --> seconds) -- thoughts on storing this as an integer instead?